europace_security.yaml

```diff
@@ -126,6 +126,9 @@ flows:
       profile: |
         ## Benutzeprofil lesen
         Profildaten (Vor- und Nachname, Benutzername, E-Mail und Avatar-Bild) des angemeldeten Benutzers können abgerufen werden.
+      silent-sign-on: |
```

Comment:
Dein Vorschlag entspricht nicht unserem aktuellen Naming-Pattern, offensichtlich gibt es Ausnahmen, aber seit einiger Zeit ist irgendwie jeder Scope eine Ausnahme und ich finde das geht in die falsche Richtung. Daher die Nachfrage, kann man das sinnvoll in das gewünschte Format `domain:resource:action` ändern?

Reply:
@MikeKrueger75 Was meinst du? Eventuell könnte `partner:zugang:silent-sign-on` oder ähnlich auch passen?

Reply:
Ich finde partner:zugang:silent-sign-on sehr sprechend

Reply:
Ist geändert. Passt das so?